### ISSUE TITLE
What to do about Infusionsoft?

### ISSUE THREAD

**Issue by joshfeck** [eventespresso/eea-importer#61]
It appears that if you import registrations, everything gets added to Infusionsoft if the IS plugin is activated. Is that something we want to happen?

If so, should be aware of any API call limitations?

**Comment by mnelson4** [comment]
Sorry I can't find any documentation on rate limiting with "Keap" (infusionsoft rebranded).
IMO we could notify users through some documentation and see what they say (if anything). I'm guessing they already have this info in Infusionsoft.
If you'd like to add a setting to control whether to sync to infusionsoft or not, that wouldn't be too hard

**Comment by joshfeck** [comment]
We'll probably need to test this then. Like what will happen if you import 100 attendees, will the process maybe time out?

Also, what exactly will happen if there's an existing contact in infusionsoft that matches a newly imported attendee record? Will a new contact be added in infusionsoft or will the existing content be updated?

**Comment by mnelson4** [comment]
ya good idea, I'll give it a whirl.

> Also, what exactly will happen if there's an existing contact in Keap that matches a newly imported attendee record

I'll test, but from the code it appears first name, last name, and email are the criteria for a match. So if those match what's already in Keap, that existing Keap contact will be updated with any other info that makes it onto the EE attendee.

**Comment by mnelson4** [comment]
I gave it a whirl with this CSV file:
[tons-of-attendees-with-custom-questions.zip](https://github.com/eventespresso/eea-importer/files/3315337/tons-of-attendees-with-custom-questions.zip)

It has Sally and Stu in there, and I see no duplicate accounts in Keap were created for them (a new order was added for them though, which is technically correct, as EE transactions correspond to Keap orders, but I don't know how users will see it). 

The only issue is that it obviously slows down the import significantly (like probably half as slow). So I created https://github.com/eventespresso/eea-importer/pull/63 to help with that.

Other than that, I'm inclined to maybe document that contacts are still sync'd to Keap in the help tab, but otherwise wait for feedback.
